lib/rules/prefer-find-by.ts

```diff
@@ -26,23 +26,26 @@ export default ESLintUtils.RuleCreator(getDocsUrl)<Options, MessageIds>({
     messages: {
       preferFindBy: 'Prefer {{queryVariant}}{{queryMethod}} method over using await {{fullQuery}}'
     },
-    fixable: null,
+    fixable: 'code',
     schema: []
   },
   defaultOptions: [],
 
   create(context) {
+    const sourceCode = context.getSourceCode();
 
-    function reportInvalidUsage(node: TSESTree.CallExpression, { queryVariant, queryMethod, fullQuery }: { queryVariant: string, queryMethod: string, fullQuery: string}) {
+    function reportInvalidUsage(node: TSESTree.CallExpression, { queryVariant, queryMethod, fullQuery, callArguments, caller }: { queryVariant: string, queryMethod: string, fullQuery: string, callArguments: TSESTree.Expression[], caller?: string }) {
       context.report({
         node,
         messageId: "preferFindBy",
         data: { queryVariant, queryMethod, fullQuery },
+        fix(fixer) {
+          const newCode = `${caller ? `${caller}.` : ''}${queryVariant}${queryMethod}(${callArguments.map((node) => sourceCode.getText(node)).join(', ')})`
```

Comment:
Damn, even checking the test I can't figure out how this replaces `waitFor` + `getBy` with `findBy`

Reply:
Do you want me to write a suggestion for this jsdoc? We need something simple, not elaborated description necessary here.

Reply:
As you want. I can do it, but I will probably do it in the weekend as I have some work to do 😄

Reply:
No problem, there is no rush around this!

Reply:
Done!
Hovering the method shows the docs:

![image](https://user-images.githubusercontent.com/352474/84548848-fd428880-acdc-11ea-8ccf-5f67957d55c9.png)

Reply:
Awesome! It wasn't even necessary to re-add the type, but thanks!